examples/07_Delegation/02_DelegatedProperties.md

```diff
@@ -38,12 +33,12 @@ fun main() {
 
 </div>
 
-1. Delegated property `p` of type `String`
-2. Delegation methods. For immutable property only `getValue` is required.
+1. Delegating property `p` of type `String` to the instance of class `Delegate`. The delegate object is defined after the `by` keyword.
+2. Delegation methods. The signatures of these methods are always as shown in the example. Implementations may contain any steps you need. For immutable properties only `getValue` is required.
 
-### Standard delegates 
+### Standard Delegates 
 
-Kotlin standard library contains bunch of useful delegates, like `lazy`, `observable`, etc.
+Kotlin standard library contains bunch of useful delegates, like `lazy`, `observable`, etc. You may use them as is.
```

Comment:
The Kotlin standard library contains a bunch of useful delegates, like `lazy`, `observable`, etc. You may use them as is.